Simple.java

```diff
@@ -0,0 +1,43 @@
+import java.io.*;
+import java.util.*;
+
+class Simple {
+    static HashMap<String, Integer> countMap = new HashMap<>();
+
+    public static void main(String[] args) throws IOException {
+        String str;
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
+        // read from stdin
+        while ((str = br.readLine()) != null) {
+            // count.
+            var st = new StringTokenizer(str.toLowerCase(), " ");
+            while (st.hasMoreTokens()) {
+                String e = st.nextToken();
+                countMap.put(e, countMap.getOrDefault(e, 0) + 1);
+            }
+        }
+        // once all counted.
+        countMap.entrySet().stream()

```

Comment:
Thanks! This seems a lot of work just to print out the values. Seeing we're using streams, how about this simplification (sent to me by KHRoN via email):

```java
        counts.entrySet().stream()
                .sorted(Entry.comparingByValue(Comparator.reverseOrder()))
                .forEach(e -> System.out.printf("%s %d\n", e.getKey(), e.getValue()));
```